### DISCUSSION TITLE
Control silence length after commas or full stops?

### DISCUSSION THREAD

**Discussion by ignacioazzi** [jing332/tts-server-android#227] [Q&A]
Hello, I was wondering whether it's possible to add pauses of custom length after commas and full stops. I am hearing a book and I need time to understand each sentence.

Thx you!

**Comment by chungusg** [comment]
I'd also like to know this! As well as being able to add longer pauses between paragraph breaks as well, to help understand dialogue and subject changes better. 
There are some examples of code on how to add different voices for different speakers in dialogue, but it's too advanced for my poor skill, and also in Chinese 😩
